.circleci/config.yml

```diff
@@ -166,7 +166,7 @@ jobs:
         working_directory: ~/repo
         docker:
             - image: nikolaik/python-nodejs:python3.7-nodejs8
-            - image: 0xorg/ganache-cli:2.2.2
+            - image: 0xorg/ganache-cli:4.4.0-beta.1
```

Comment:
Was this intentional?

Reply:
Yes.  Adding in the test of `build_transaction()` ran me into a problem with Web3.py trying to do something with `eth_chainId`, and that wasn't a supported RPC method in the version of ganache-cli used in the 2.2.2 image.  The 4.4.0-beta.1 image has the updated ganache-cli, as well as the snapshot needed to support the tests, so I ran with it.